tests/ci/run_fips_tests.sh

```diff
@@ -10,6 +10,15 @@ fips_build_and_test
 echo "Testing AWS-LC in FIPS release mode."
 fips_build_and_test -DCMAKE_BUILD_TYPE=Release
 
+echo "Testing shared AWS-LC in FIPS debug mode."
+fips_build_and_test -DBUILD_SHARED_LIBS=1
+
+# FIPS build in release mode is disabled for GCC due to some build issues relating to '-O3'.
+if [[ "${CC}" == 'clang'*  ]]; then
```

Comment:
This is tested with only clang 7 or more clang versions?

Reply:
Only Clang-7. I don't plan to add more compiler because this is POC. Final supported compiler will be addressed in CryptoAlg-714

Reply:
Ack, thanks